app/src/main/java/org/piramalswasthya/sakhi/ui/home_activity/HomeActivity.kt

```diff
@@ -203,10 +218,89 @@ class HomeActivity : AppCompatActivity() {
         }
     }
 
+   private fun displaychatdialog() {
+
+        val dialog = BottomSheetDialog(this)
+
+        // on below line we are inflating a layout file which we have created.
+        val view = layoutInflater.inflate(R.layout.bottomsheet_chat_window, null)
+
+        val web = view.findViewById<WebView>(R.id.webv)
+        val progress = view.findViewById<ProgressBar>(R.id.progressBarv)
+
+
+
+// Enable JavaScript
+        web.settings.javaScriptEnabled = true
+        web.settings.javaScriptCanOpenWindowsAutomatically = true
+        web.isVerticalScrollBarEnabled = true
+
+
+
+
+// Load URL
+        web.loadUrl("https://aibot.helloyubo.com/piramal_bot_rag")
```

Comment:
Please don't hardcode this. It must be a build variable.
Also provide option to turn off chat for certain builds.

Reply:
i have done this